webpack.config.js

```diff
@@ -0,0 +1,27 @@
+const path = require('path');
+
+module.exports = {
+  entry: './src/index.jsx',
+  mode: 'development',
+  module: {
+    rules: [
+      {
+        test: /\.(js|jsx)$/,
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            presets: ['@babel/preset-env'],
```

Comment:
I set mine up putting both presets into the array and not configuring babel as we discussed this is an option for webpack 5 but not for 4 but something to note.

Reply:
Updated.